### DISCUSSION TITLE
Is there any way to tell the FileManager to use a middleware (e.g. axios) for authentication

### DISCUSSION THREAD

**Discussion by MoritzTheile** [OpusCapita/filemanager#323] [Q&A]
After implementing a OpusCapita Spring Boot interface ([see other discussion](https://github.com/OpusCapita/filemanager/discussions/321#discussioncomment-1323177)), I was wondering if we can utilize our axios middleware for authentication. So each http requests from the FileManager would send along the Authentication Bearer in the header, injected by axios.

Maybe I missed the documentation where this is described.

**Comment by estambakio-sc** [comment]
Do you mean adding authentication capability to client (in-browser) part? This requires adding custom logic to connector implementation (see for example [Google Drive connector](https://github.com/OpusCapita/filemanager/blob/master/packages/connector-google-drive-v2/src/api.js#L320)).
